students/NatalieRodriguez/Lesson03/slices.py

```diff
@@ -0,0 +1,74 @@
+#Lesson 03: Slices
+#Natalie Rodriguez
+#3/14/2018
+
+
+#Test Sequences to Use
+dog_list = ['Virgil', 'River', 'Kibson', 'Dexter', 'China', 'Lucy', 'Percy']
+bird_list = ['Bluejay', 'Sparrow', 'House Finch', 'Cardinal', 'Nuthatch', 'Tufted Titmouse']
+number_list = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
+
+#________________________________________________________________________
+#Exchange the first item in the sequence with the last item in the sequence using slices.
+def first_last_exchanged(dog_list):
+    print("Original Sequence:",dog_list)
+    dog_list[0], dog_list[-1] = dog_list[-1], dog_list[0]
+    return dog_list
```

Comment:
how can you solve it using slicing only?

`[-1]` / `[0]` are indexing
`[start:stop:step]` is slicing (hint: its a one liner)